modules/ts-capi-sync-list-duplicate-resources.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cluster_api_machine_management/cluster-api-troubleshooting.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ts-capi-sync-list-duplicate-resources_{context}"]
+= Duplicated machine set and machine resources
+
+On clusters that support migrating Machine API resources to Cluster API resources, some resources seem to have duplicate instances in the output of {oc-first} commands that list resources and in the {product-title} web console.
+
+Cause::
+
+When you install an {product-title} cluster that uses the default configuration options, the installation program provisions the following infrastructure resources in the `openshift-machine-api` namespace:
++
+--
+* One control plane machine set that manages three control plane machines.
+* One or more compute machine sets that manage three compute machines.
+* One machine health check instance.
+--
++
+On clusters that support migrating Machine API resources to Cluster API resources, a two-way synchronization controller creates the following Cluster API resources in the `openshift-cluster-api` namespace:
++
+--
+* One cluster resource.
+* One provider-specific infrastructure cluster resource.
+* One or more machine templates that correspond to compute machine sets.
+* One or more compute machine sets that manage three compute machines.
+--
++
+These Cluster API resources have the same names as their counterparts in the `openshift-machine-api` namespace.
+
+Consequence::
+
+Due to this behavior, instances of machine set and machine resources that seem to be duplicates appear in the output of `oc` commands that list resources and in the {product-title} web console.
```

Comment:
I can't say I've used the web console recently.... but is it possible we could improve the UX to potentially avoid any confusion? Bring in a tool tip or modal?

Reply:
Discussed in Slack, but to close the loop: if we want to do this, the next step is to reach out to the console team about it. I agree that an in-situ tip would probably be helpful for stuff like this :)

Reply:
followed your thread to https://redhat-internal.slack.com/archives/C6A3NV5J9/p1751965661343749